hypothesis-python/tests/cover/test_settings.py

```diff
@@ -81,6 +95,10 @@ def test_respects_none_database():
     assert settings(database=None).database is None
 
 
+@pytest.mark.skipif(
+    settings._current_profile != "default",
+    reason="explicitly checks against settings.get_profile('default').max_examples",
+)
```

Comment:
I think we could un-skip if we changed `assert settings().max_examples == original_default` to `original_default = settings().max_examples`?

We could also keep the assertion, extended to `... or settings._current_profile != "default"`.

Reply:
I didn't do this because it conflicts with `test_load_profile`, which does want `settings.get_profile("default").max_examples`. I've moved `original = settings().max_examples` inside the test here, which slightly weakens it (no longer checking that another test didn't modify settings since startup), but acceptably so, especially since it gets rid of the skip.